apps/prairielearn/elements/pl-file-preview/pl-file-preview.js

```diff
@@ -133,10 +133,6 @@
                     import('dompurify'),
                     import('notebookjs'),
                   ]).then(async ([Marked]) => {
-                    // Showdown has a small bug that doesn't allow it to be loaded dynamically.
-                    // This PR will fix it: https://github.com/showdownjs/showdown/pull/1017
-                    // Since the PR could take two weeks or two months, let's used Marked for now
-                    // and get this feature deployed.
                     nb.markdown = Marked.marked.parse;
```

Comment:
I haven't handled mathjax in the preview of notebook files, is it needed?

Reply:
It's already handled here, right?

https://github.com/PrairieLearn/PrairieLearn/blob/ad778ab3370b197aa413f2c69684f66e9f5c45b8/apps/prairielearn/elements/pl-file-preview/pl-file-preview.js#L150-L151

Reply:
Ahhh, I see. Check `demo/workspace/jupyterlab` in the example course, it includes the following Markdown cell:

```
The Fibonacci numbers are 1, 1, 2, 3, 5, 8, ..., where each number is the sum of the two before it. These are numbered $F_1, F_2, \ldots$, and

$$F_n = F_{n-1} + F_{n-2}.$$
```

I think the underscores in there could be considered "math that can be interpreted as markdown".

Reply:
It was indeed needed, changing the math to add some spaces caused `_` to be interpreted as markdown. Updated in 03e677d4f7cb5e4aa7b8bcb137bf483f6051771c.